src/ui/popup/settings-page/enable-browser-theme.tsx

```diff
@@ -0,0 +1,20 @@
+import {m} from 'malevic';
+import CheckButton from '../check-button';
+import {ViewProps} from '../types';
+
+export default function EnableCustomBrowserTheme(props: ViewProps) {
```

Comment:
I think `ChangeBrowserTheme` has the same meaning, but is shorter. Please rename it in all files.

Also rename this file to `change-browser-theme.tsx`.